FEXCore/Source/Interface/Core/JIT/JITClass.h

```diff
@@ -61,6 +62,15 @@ class Arm64JITCore final : public CPUBackend, public Arm64Emitter {
   const bool HostSupportsRPRES {};
   const bool HostSupportsAFP {};
 
+  struct RestartOptions {
+    FEXCore::LongJump::JumpBuf RestartJump;
+    bool NeedsLongJumps {};
+    enum class RestartOptionControl : uint64_t {
```

Comment:
I think we're referring to this as `RestartOptions::RestartOptionControl` everywhere, so might as well drop the prefix:
```suggestion
    enum class Control : uint64_t {
```